app/src/main/java/at/aau/edu/wizards/gameModel/GameModel.kt

```diff
@@ -40,6 +43,7 @@ class GameModel(private val viewModel: GameBoardViewModel?) {
         ))
     }
 
+    // FIXME meaningful message content (eg, GameCommand), or message parsing in dedicated class for maintainability
```

Comment:
Message parsing is already added with GameModelDebugger or am I missing something?